geomesa-cassandra/geomesa-cassandra-tools/src/main/scala/org/locationtech/geomesa/tools/cassandra/commands/CommandWithCassandraDataStore.scala

```diff
@@ -0,0 +1,33 @@
+package org.locationtech.geomesa.tools.cassandra.commands
```

Comment:
Yep, you can rip if off any other file, just make it look the same. If you use intellij to copy it over it may auto-indent all but the first line which'll make the license check fail.